src/components/Admin/Security/SecurityManagementContents.tsx

```diff
@@ -1,25 +1,27 @@
 import React, { Fragment, useMemo, useState } from 'react';
-import PropTypes from 'prop-types';
-import { withTranslation } from 'react-i18next';
-
+// import PropTypes from 'prop-types';
 import { TabContent, TabPane } from 'reactstrap';
+import { useTranslation } from '~/i18n';
+
+import LdapSecuritySetting from '~/client/js/components/Admin/Security/LdapSecuritySetting';
+import LocalSecuritySetting from '~/client/js/components/Admin/Security/LocalSecuritySetting';
+import SamlSecuritySetting from '~/client/js/components/Admin/Security/SamlSecuritySetting';
+import OidcSecuritySetting from '~/client/js/components/Admin/Security/OidcSecuritySetting';
+import SecuritySetting from '~/client/js/components/Admin/Security/SecuritySetting';
+import BasicSecuritySetting from '~/client/js/components/Admin/Security/BasicSecuritySetting';
+import GoogleSecuritySetting from '~/client/js/components/Admin/Security/GoogleSecuritySetting';
+import GitHubSecuritySetting from '~/client/js/components/Admin/Security/GitHubSecuritySetting';
+import TwitterSecuritySetting from '~/client/js/components/Admin/Security/TwitterSecuritySetting';
+import FacebookSecuritySetting from '~/client/js/components/Admin/Security/FacebookSecuritySetting';
+import ShareLinkSetting from '~/client/js/components/Admin/Security/ShareLinkSetting';
+
+import CustomNav from '~/client/js/components/CustomNavigation/CustomNav';
+
+type Props = {
+};
 
-import LdapSecuritySetting from './LdapSecuritySetting';
-import LocalSecuritySetting from './LocalSecuritySetting';
-import SamlSecuritySetting from './SamlSecuritySetting';
-import OidcSecuritySetting from './OidcSecuritySetting';
-import SecuritySetting from './SecuritySetting';
-import BasicSecuritySetting from './BasicSecuritySetting';
-import GoogleSecuritySetting from './GoogleSecuritySetting';
-import GitHubSecuritySetting from './GitHubSecuritySetting';
-import TwitterSecuritySetting from './TwitterSecuritySetting';
-import FacebookSecuritySetting from './FacebookSecuritySetting';
-import ShareLinkSetting from './ShareLinkSetting';
-
-import CustomNav from '../../CustomNavigation/CustomNav';
-
-function SecurityManagementContents(props) {
-  const { t } = props;
+const SecurityManagementContents = (props: Props): JSX.Element => {
```

Comment:
```suggestion
const SecurityManagementContents:FC<Props> = (props: Props) => {
```
こちらの方が良さそうです
参考：https://qiita.com/Takepepe/items/0bbe4ab6ff4858a4e186#-winners

Reply:
なるほど、。
 `SecurityManagementContents = (props: Props)`
に関して、引数の型推測では
`SecurityManagementContents = (props: Props): JSX.Element`
のことを指すんですね。
知らなかったです。

また、定義した`SecurityManagementContents:FC `に型をつけていませんでしたので、そちらも修正しました。